nym-wallet/src/components/Accounts/AccountOverview.tsx

```diff
@@ -6,7 +6,7 @@ import { AccountAvatar } from './AccountAvatar';
 export const AccountOverview = ({ account, onClick }: { account: AccountEntry; onClick: () => void }) => (
   <Button
     startIcon={<AccountAvatar name={account.id} small />}
-    sx={{ color: 'text.primary' }}
+    sx={{ color: 'text.primary', fontSize: 14 }}
```

Comment:
This is ok for now but in the future we need to come back and fix this up. Ideally these values should come from somewhere in the theme which will make things more manageable and consistent.